src/windows/views/tutorial.py

```diff
@@ -148,12 +131,20 @@ def __init__(self, id, text, arrow, manager, *args):
         hbox = QHBoxLayout()
         hbox.setContentsMargins(20, 10, 0, 0)
 
+        # Close action
+        self.close_action = QAction(_("Hide Tutorial"))
```

Comment:
self.close_action = QAction(_("Hide Tutorial"), self)

Reply:
Probably due to my older version of Qt, I'm guessing

Reply:
Yeah, I guess this note applies to _both_ of [these constructors](https://doc.qt.io/qt-5/qaction.html#QAction) (not that they make that remotely clear.)
> **Note:** The _parent_ argument is optional since Qt 5.7.